.github/CONTRIBUTING.md

```diff
@@ -73,4 +72,4 @@ language we use. View them [here](https://github.com/fheng/best_practice).
 
 * [General GitHub documentation](http://help.github.com/)
 * [GitHub pull request documentation](https://help.github.com/articles/about-pull-requests/)
-* [Read the Issue Guidelines by @necolas](https://github.com/necolas/issue-guidelines/blob/master/CONTRIBUTING.md) for more details
\ No newline at end of file
+* [Read the Issue Guidelines by @necolas](https://github.com/necolas/issue-guidelines/blob/master/CONTRIBUTING.md) for more details
```

Comment:
This is probably fine now but I think we'll update it to link to some content around how we manage github issues coming out of the discussion at the community meeting earlier today.

Reply:
Ok, will wait for that meeting to update it and merge this pr for now